lib/salus/report.rb

```diff
@@ -124,6 +126,8 @@ def to_json
 
     def to_sarif
       Sarif::SarifReport.new(@scan_reports).to_sarif
+    rescue Sarif::SarifReport::SarifInvalidFormatError => e
```

Comment:
Could there be any other types of exceptions from `Sarif::SarifReport.new(@scan_reports).to_sarif`?